wvd-templates/wvd-scaling-script/CreateOrUpdateAzAutoAccount.ps1

```diff
@@ -46,8 +46,10 @@ if (!$UseRDSAPI) {
 	$WebhookName += 'ARMBased'
 }
 
-# Set the ExecutionPolicy
-Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope CurrentUser -Force -Confirm:$false
+# Set the ExecutionPolicy if not being ran in CloudShell as this command fails in CloudShell
+if (-not ($env:POWERSHELL_DISTRIBUTION_CHANNEL -eq 'CloudShell') {
```

Comment:
`CreateOrUpdateAzLogicApp.ps1` also needs to be updated with this change.

Reply:
Added the same change to both scripts